doc/policy_tutorial/POLICY_RUN_LOCALLY.md

```diff
@@ -83,7 +89,8 @@ Now you have another interactive bash shell in the APIcast development container
 In the container issue the following HTTP request:
 
 ```shell
-$ curl localhost:8080
+# user_key is a required paramerter for the echoapi backend
+$ curl -H "Host: one" "http://localhost:8080/?user_key="
```

Comment:
nit: I would use `--resolve` curl option

```
curl --resolve one:8080:127.0.0.1 -v "http://one:8080/?user_key"
```

Up to you, not really requesting a change here.

Reply:
Interesting not an option I have tried before with curl. 
A kind of dns resolver for curl .
Does it make it more or less understandable :thinking: 
Think I will leave as is